kore/src/Kore/Repl.hs

```diff
@@ -168,14 +167,14 @@ runRepl axioms' claims' logger replScript replMode outputFile = do
             , labels     = Map.empty
             , aliases    = Map.empty
             , koreLogOptions =
-                Logger.KoreLogOptions
-                    { logType = Logger.LogStdErr
+                Log.KoreLogOptions
```

Comment:
I think this is equal to the default value for `KoreLogOptions`. Do we want to use `def` here instead?